downloaders/dumps_downloader.py

```diff
@@ -0,0 +1,76 @@
+import urllib.request
```

Comment:
In general - let's follow this [style convention](https://github.com/src-d/guide/blob/master/engineering/conventions/python.md)